.github/workflows/test.yml

```diff
@@ -38,19 +38,35 @@ jobs:
     strategy:
       matrix:
         include:
-          - wagtail-version: "Wagtail>=3.0,<4.0"
-            django-version: "Django>=4.0,<5.0"
-            python-version: 3.9
+          # Django 5.x supports Python 3.10, 3.11, and 3.12
+          # https://docs.djangoproject.com/en/5.0/releases/5.0/#python-compatibility
+          - django-version: "Django>=5.0,<6.0"
```

Comment:
Suggestion: Should we explicitly limit to 5.0.x only? 5.1 isn't going to be for a while, but still.

Reply:
You're right! Addressed in [3e9ab2a](https://github.com/torchbox/wagtail-bucketav/pull/6/commits/3e9ab2a1f72af7931b787744741041623d2150bb)